src/script/components/panel/userActions.js

```diff
@@ -136,7 +136,7 @@ ko.components.register('user-actions', {
           return isNotMe && isNotConnectedUser && canConnect;
         },
         item: {
-          click: () => actionsViewModel.sendConnectionRequest(user).then(() => onAction(Actions.SEND_REQUEST)),
+          click: () => actionsViewModel.sendConnectionRequest(user, true).then(() => onAction(Actions.SEND_REQUEST)),
```

Comment:
Just for the record: I might create a follow up PR that removes the fact that the `ConnectionRepository` knows how to control the UI!!!